src/styles/styles.js

```diff
@@ -325,6 +325,7 @@ const styles = {
     sidebarHeaderTop: {
         paddingHorizontal: 20,
         paddingVertical: 12,
+        minHeight: 64,
```

Comment:
This is probably not necessary after the icon is put into the new spot.

Reply:
removed `minHeight: 64`